src/sagemaker/model.py

```diff
@@ -651,7 +651,9 @@ def compile(
         job_status = self.sagemaker_session.wait_for_compilation_job(job_name)
         self.model_data = job_status["ModelArtifacts"]["S3ModelArtifacts"]
         if target_instance_family is not None:
-            if target_instance_family.startswith("ml_"):
+            if target_instance_family == "ml_eia2":
+                LOGGER.info("You are using target device ml_eia2...")
```

Comment:
what is this for?

Reply:
Our trt target device name is "ml_eia2", we don't need neither set the _compilation_image_uri which is just for neo and inferentia nor set the model as compiled since in our case the compiled model uses the same deploy workflow as what the uncompiled model does.